script.js

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env node
+
+"use strict";
+const _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) { return typeof obj; } : function (obj) { return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype ? "symbol" : typeof obj; };
+
+const path = require('path'),
+    fs = require('fs');
```

Comment:
Maybe `const fs = require('fs');` would look better? It looks a bit out of place right now, since we use a different code style across the file.

Reply:
Fixed